masterlist.yaml

```
@@ -22689,16 +22689,28 @@ plugins:
         util: 'SSEEdit v4.0.4b'
 
   - name: 'The Great City of Solitude.esp'
-    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/22243/' ]
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/22243/'
+        name: 'The Great City of Solitude SSE'
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/72117/'
+        name:  'Rob''s Bug Fixes - TGC Solitude'
```

There's an extra space here:
```suggestion
        name: 'Rob''s Bug Fixes - TGC Solitude'
```